css/media.css

```diff
@@ -0,0 +1,429 @@
+/*@media only screen and (min-width: 768px) {
+	.product-tile {
+		width: 50%;
+	}
+}*/
+
+/*@media only screen and (min-width: 1000px) {
+	.product-tile {
+		width: 25%;
+	}
+}*/
+
+/*@media only screen and (max-width: 1170px) {
+
+}*/
+
+@media only screen and (max-width: 1024px) {
+	.header-content ,
+	.poster-content ,
+	.product-content {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457137)